### PR TITLE
809 create poi

### PR DESCRIPTION
**Pull request by Kai-ros**
The route needs to be fixed, otherwise everything is in working order.

### CODE REVIEW

**Comment by Kai-ros** [comment]
Getting a 400 Bad Request error with this route:
image

File: src/components/POI/POI.tsx [hunk 1]
```diff
@@ -164,6 +164,19 @@ const POI = () =>{
...
+    // remove this once login is implemented
+    let createdBy : User = {
+        FirstName : 'Marc',
+        LastName : 'Miller',
+        LogonName : 'marc.miller@slalom.com',
+        EmailAddress : 'marc.miller@slalom.com',
+        CellNumber : '206-962-1616'
+    }
+    const dateTime = () =>{
+        var time = new Date();
```
Comment (Kai-ros) [inline src/components/POI/POI.tsx:176]: Need to use let as above, var shouldn't be used.

File: src/API/Contracts.ts [hunk 2]
```diff
@@ -34,5 +34,15 @@ export interface POI {
     AssociatedTasks : object,
     FullName : string,
     LastNameFirst : string,
-    Created : string
+    Created : string,
+    CreatedBy: User
 }
+
+export interface User {
+
```
Comment (Kai-ros) [inline src/API/Contracts.ts:42]: Remove this extra line please.

**Review by samlislalom — APPROVED** [review]

File: src/API/Contracts.ts [hunk 3]
```diff
@@ -1,6 +1,4 @@
-
-
-export interface CreateCaseObject { 
+export interface Case { 
     NickName?: string,
     CaseNumber: string,
     CreatedBy?: {
```
Comment (SlalomMarc) [inline src/API/Contracts.ts:4]: Not sure why CreatedBy is nullable.  We always need to know who created items in this.

File: src/API/Endpoints.ts [hunk 4]
```diff
@@ -1,6 +1,10 @@
 
-const baseURL = 'https://virtualcommandpost-mmtest.azurewebsites.net/api/';
+const baseRoute = 'https://sdcs-vcp-api-dev.azurewebsites.net/api/';
```
Comment (SlalomMarc) [inline src/API/Endpoints.ts:2]: These values should be in some kind of configuration so SDCS Build Pipelines will deploy properly.  (this URL is the Slalom Development Sandbox URL and SDCS will not be using that)

**Review by SlalomMarc — CHANGES_REQUESTED** [review]
Couple of minor-ish things to look at here.  Definitely not nits.

**Pull request merged**